tools/clang/lib/SPIRV/LowerTypeVisitor.cpp

```diff
@@ -87,6 +87,15 @@ bool LowerTypeVisitor::visitInstruction(SpirvInstruction *instr) {
           lowerType(debugQualType, instr->getLayoutRule(),
                     /*isRowMajor*/ llvm::None, instr->getSourceLocation());
       debugInstruction->setDebugSpirvType(spirvType);
+    } else if (auto *debugGlobalVar =
+                   dyn_cast<SpirvDebugGlobalVariable>(instr)) {
+      auto *varType = dyn_cast<SpirvPointerType>(
+          debugGlobalVar->getVariable()->getResultType());
+      auto *actualType = varType->getPointeeType();
+      debugGlobalVar->setDebugSpirvType(actualType);
+      if (auto *structType = dyn_cast<StructType>(actualType))
+        lowerDebugTypeComposite(structType, structType->getFields(), false,
```

Comment:
Why do you need to pass `structType->getFields()` to the function?

`lowerDebugTypeComposite` has access to `structType` and can call `structType->getFields()` itself.